pyproject.toml

```diff
@@ -1,10 +1,99 @@
+[project]
+name = "rockcraft"
+dynamic = ["version"]
+description = "Create rocks"
+license = {file = "LICENSE"}
+readme = "README.rst"
+requires-python = ">= 3.10"
+authors = [
+    { name = "Canonical Ltd.", email = "snapcraft@lists.snapcraft.io" },
+]
+classifiers = [
+    "Development Status :: 5 - Production/Stable",
+    "Intended Audience :: Developers",
+    "License :: OSI Approved :: GNU General Public License v3 (GPLv3)",
+    "Operating System :: MacOS :: MacOS X",
+    "Operating System :: POSIX :: Linux",
+    "Programming Language :: Python :: 3",
+    "Programming Language :: Python :: 3.10",
+]
+dependencies = [
+    "craft-application>=4.0.0",
+    "craft-archives>=2.0.0",
+    "craft-cli",
+    "craft-parts>=2.1.2",
+    "craft-platforms~=0.3",
+    "craft-providers>=2.0.4",
+    "overrides",
+    "spdx-lookup",
+    "tabulate>=0.8.10",
+]
+
+[project.scripts]
+rockcraft = "rockcraft.cli:run"
+
+[project.optional-dependencies]
+dev = [
+    "black",
+    "canonical-sphinx[full]",
+    "codespell",
+    "coverage",
+    "mccabe<0.7.0",
```

Comment:
Should this `mccabe` import preserve the comment from the old `setup.cfg`? It was "`# to resolve version conflict`". It seems like it was added 2 years ago, so it may not be relevant anymore, but I'd like to make sure.

Reply:
I'll do ya one better: this should have been removed when we removed flake8, so I've just tossed it now :-)